dashboard/templates/session.html

```diff
@@ -70,8 +70,8 @@ <h2>Stats</h2>
                   <span class="value">{{ session.client_bytes_sent }} / {{ session.client_bytes_received }}</span>
                 </li>
                 <li>
-                  <span class="name">Latency</span>
-                  <span class="value">N/A</span>
+                  <span class="name">Client Country</span>
+                  <span class="value">{{ session.client_country }}</span>
```

Comment:
Shouldn't we show "N/A" in case it's empty, as in other places?

Reply:
fixed